app/views/issues/_attributes.rhtml

```diff
@@ -1134,6 +1134,10 @@ Event.observe(window, 'load', function() {
       adjustDueDate()
     else if (label == 'New')
       setDueDate(null)
+    else if (label == 'Dispatched') {
+      const dispatchedCheckbox = findInput("Dispatched", "input[type=checkbox]");
+      dispatchedCheckbox.checked = true;
```

Comment:
you might want to check if `dispatchedCheckbox` is not null/undefined before checking the box to be safe